app/assets/javascripts/dashboard/subsidies.js.coffee

```diff
@@ -40,9 +40,11 @@ $(document).ready ->
 
   $(document).on 'click', '.delete_subsidy_button', ->
     subsidy_id = $(this).data('subsidy-id')
-    $.ajax
-      type: 'DELETE'
-      url: "/dashboard/subsidies/#{subsidy_id}"
+
+    if confirm("Are you sure?")
```

Comment:
Can you make this confirmation message a little more descriptive? As in "Are you sure that you would like to delete this subsidy?"